docs/3.tools/welcome.md

```diff
@@ -77,6 +77,14 @@ Welcome! If you want to integrate a deployed smart contract into your app you ha
   </div>
 </div>
 
+## What's the timeline/plan for supporting general widely used programming languages so devs can just code in whatever they're comfortable working with?
```

Comment:
This can go into develop/whatiscontract ,  there is a section about programming languages.